### ISSUE TITLE
Collections did not get unarchived on upgrading

### ISSUE THREAD

**Issue by g-macd** [a85/POSTMan-Chrome-Extension#7476]
**Describe the bug**
I am an Admin for our Team. We were previously a Free team, and our Collections were Archived. We have since upgraded to Pro, but i cannot get to the /usage/archive page to un-archive our collections. When we try, we receive error message

_**You don't have permissions to view this page**
Only users with the plan can view this page._

**To Reproduce**

**Expected behavior**
View Archived collections and un-archive them

**Screenshots**

**App information (please complete the following information):**
 - App Type [e.g. Chrome App, Native App]
 - Postman Version [e.g. 7.0.0]
 - OS: [e.g. macOS High Sierra 10.13.2]

**Additional context**

**Comment by sankalp0o** [comment]
Hey @g-macd , thanks for writing to us. When you upgraded, all archived collections would've been unarchived and restored to respective workspaces. Is there a collection that was archived but you can't find now?

The messaging on the page is misleading and we will correct that.

**Comment by g-macd** [comment]
Hi @sankalp0o, our collections weren't unarchived, and the postman app is still saying "Some collections were archived because you’ve reached the shared requests limits."

Our team is https://asp-it-devops.postman.co, are you able to check & unarchive them?

**Comment by sankalp0o** [comment]
Thanks for getting back! This looks like a bug, we'll take a look and fix this as soon as possible.

**Comment by g-macd** [comment]
Hi @sankalp0o, thanks for looking in to this for us - have you been able to make any progress?

**Comment by sankalp0o** [comment]
Hey @g-macd , we have identified the problem and are working towards fixing it. Our highest priority right now is to unblock you and your team. Hopefully this will get resolved within the next week. Sorry for the inconvenience!

**Comment by ash007-postman** [comment]
@sankalp0o Another user (Postman enterprise) facing the same issue, more details can be found at Zendesk(40309)

**Comment by laNds** [comment]
> @sankalp0o Another user (Postman enterprise) facing the same issue, more details can be found at Zendesk(40309)

I'm the user, thanks for the reply, i'll follow this issue.
@sankalp0o is the unblock process is only for @g-macd or will work for our team too ?

**Comment by g-macd** [comment]
I managed to get this working by fully logging out of my account (in the Browser and in the Windows app), and logging back in. It seemed like doing this caused a full re-sync of all workspaces & collections.

**Comment by laNds** [comment]
@g-macd : wow thanks, that work for me too.

**Comment by sankalp0o** [comment]
Glad it helped! Were you able to recover all the collections @g-macd @laNds? Or are there still some that are archived?

**Comment by laNds** [comment]
@sankalp0o : yeah, we got all of our collections back 👍 .

**Comment by sankalp0o** [comment]
Closing this as duplicate of #6027
